History.txt

```diff
@@ -1,9 +1,34 @@
-Changes between FreeRTOS V10.4.6 and TBD
+Changes between FreeRTOS V10.4.6 and FreeRTOS V10.5.0 released September 16 2022
 
 Documentation and download available at https://www.FreeRTOS.org/
 
+	+ ARMv7-M and ARMv8-M MPU ports: It is possible for a third party that

```

Comment:
The history file should make it clear these issues applied to the prior version.  So "it was possible", rather than "is possible", as a minimum.